lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js

```diff
@@ -51,15 +51,17 @@ class ExternalAnchorsUseRelNoopenerAudit extends Audit {
 
     const pageHost = new URL(artifacts.URL.finalUrl).host;
     // Filter usages to exclude anchors that are same origin
+    // TODO: better extendedInfo for anchors with no href attribute:
+    // https://github.com/GoogleChrome/lighthouse/issues/1233
     const failingAnchors = artifacts.AnchorsWithNoRelNoopener.usages
-      .filter(anchor => new URL(anchor.href).host !== pageHost)
+      .filter(anchor => anchor.href === '' || new URL(anchor.href).host !== pageHost)
       .map(anchor => {
         return {
-          url: `<a
-            href="${anchor.href}"
-            ${anchor.target ? ` target="${anchor.target}"` : ''}
-            ${anchor.rel ? ` rel="${anchor.rel}"` : ''}>...
-          </a>`
+          url: '<a' +
+              (anchor.href !== '' ? ` href="${anchor.href}"` : '') +
```

Comment:
the template string here rendered fine in the html report but took up five lines per link in the pretty print report, e.g.
```
      <a
            href="http://get.grubhub.com/?referer=seamless.com"
             target="_blank"
            >...
          </a>
```
The logic also got kind of complicated even when just dropping the href if it was `''` (since it would leave a blank line), so easier to just turn this into string concatenation.